ckanext/s3filestore/uploader.py

```diff
@@ -416,6 +419,7 @@ def get_path(self, id, filename=None):
 
         if filename is None:
             filename = os.path.basename(self.url)
+        filename = munge.munge_filename(filename)
```

Comment:
url is munged on storage so this is not required.

Reply:
It may be required if `get_path` is called from externally, eg by ckanext-archiver.